test/blackbox-tests/utils/dune_stat.ml

```diff
@@ -0,0 +1,21 @@
+open Stdune
+
+type data =
+  | Hardlinks
+  | Permissions
+
+let data_of_string = function
+  | "hardlinks" -> Hardlinks
+  | "permissions" -> Permissions
+  | s -> failwith ("invalid data " ^ s)
```

Comment:
Just because "data" could be ambiguous, and someone might try to call this on a file directly:

```suggestion
  | s -> failwith ("invalid first argument " ^ s)
```

An even more helpful result would be printing out the valid selections, but given the aim and situation of this utility, I don't think user friendliness is very important :)